swaglyrics/spotify.py

```diff
@@ -45,20 +45,24 @@ def get_info_linux():
 
 def get_info_mac():
 	from Foundation import NSAppleScript
-	apple_script_code = """
-	getCurrentlyPlayingTrack()
-	on getCurrentlyPlayingTrack()
```

Comment:
What do these changes accomplish?

Reply:
They perform the same action. but the way of getting info is changed

The reason I had to change was because , sometimes, the first code failed to get me the details.

Reply:
Did that happen specifically with Chrome or in general as well? Also, you're on a Mac?

Reply:
nope. In general for the first time.

yes I am